src/components/a-form-input/index.scss

```diff
@@ -0,0 +1,111 @@
+@import "../../styles/settings/colors";
+@import "../../styles/settings/controls";
+@import "../../styles/mixins/typo";
+@import "../../styles/mixins/control-transition";
+
+.a-form-input-wrapper {
```

Comment:
this does not fully satisfy our BEM naming convention. Sorry for have not done a proper documentation yet. Maybe call me and I will inform you. But we generally use BEM where the element is separated with `__` from the block name and modifier with `--`

Reply:
here would be i guess a-form-input the block and wrapper the element.